updated docs (#31192)
demisto/content: Packs/MicrosoftGraphDeviceManagement/ReleaseNotes/1_1_20.md

#### Integrations

##### Microsoft Endpoint Manager (Intune)
- Updated the Docker image to: *demisto/crypto:1.0.0.82686*.

> - Updated the yml file in ***msgraph-get-managed-device-by-id*** command's description, that physicalMemoryInBytes will return 0 by default.

Reviewer: ```suggestion
- Added a note regarding ***msgraph-get-managed-device-by-id*** command, that physicalMemoryInBytes is 0 by default.
```

#### Integrations

##### Microsoft Endpoint Manager (Intune)

> - Added a note in the README file that by default physicalMemoryInBytes will return 0 from the command ***msgraph-get-managed-device-by-id***,

Reviewer: The README is not relevant to the RN. 
We should:
1. Add the note from the readme also to the yml file.
2. Update the RN accordingly.

Author: done
